scripts/project.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
this defines a `scripts` project in our workspace for nx inside the `scripts` directory.

With this, we can now call: `nx test:eslint scripts` to lint our scripts, and what's cooler, we can now just "add this to the pile" by calling `nx run-many --target=test:lint` inside of our package scripts/ci stuff :)